src/ric/d_19EE0.c

```diff
@@ -0,0 +1,362 @@
+#include "ric.h"
+
+s32 D_80155EE0[] = {0x00000F80, 0x00000100, 0x00000700, 0x00000880};
+
+s16 D_80155EF0[] = {0x0400, 0x06AB, 0x0955, 0x0C00, 0x0EAB, 0x1155};
+
+SVECTOR D_80155EFC = {
+    0xFFF8,
+    0xFFB8,
+    0xFFFC,
+    0x0000,
+};
```

Comment:
```suggestion
SVECTOR D_80155EFC = { -8, -72, -4 };
```
same for the other `SVECTOR`s